portal/templates/profile_macros.html

```diff
@@ -872,7 +872,7 @@ <h4 class="modal-title">{{ _("Consent to share information") }}</h4>
                         item["filename"] = escapeHtml(item["filename"]);
                         item["document_type"] = escapeHtml(item["document_type"]);
                         item["uploaded_at"] = tnthDates.formatDateString(item["uploaded_at"], "iso");
-                        item["actions"] = '<a title="Download" href="' + '/api/user/' + String(item["user_id"]) + '/user_documents/' + String(item["id"]) + '"><i class="fa fa-arrow-down"></i></a>';
+                        item["actions"] = '<a title="Download" href="' + '/api/user/' + String(item["user_id"]) + '/user_documents/' + String(item["id"]) + '"><i class="fa fa-download"></i></a>';
```

Comment:
Could you tag "Download" with `trans` / `__()` / `i18next.t()`?

Can you use `url_for()` here?

Reply:
ivan, started to make the fix to change to url_for but then I realized the user id and the document id come from the client side - making it infeasible in this case